### ISSUE TITLE
Programming error: relation "crm_case_section" does not exist

### ISSUE THREAD

**Issue by mgielissen** [OCA/OpenUpgrade#637]
While running the script to migrate a v7 to v8 I get this error:

```
2016-09-20 08:30:19,785 11711 ERROR moto_migrated openerp.sql_db: Programming error: relation "crm_case_section" does not exist
LINE 1: UPDATE crm_case_section SET code = '_DM' WHERE code = 'DM'
               ^
, in query UPDATE crm_case_section SET code = '_DM' WHERE code = 'DM'
2016-09-20 08:30:19,786 11711 ERROR moto_migrated OpenUpgrade: sales_team: error in migration script /var/tmp/openupgrade/8.0/addons/sales_team/migrations/8.0.1.0/pre-migration.py: relation "crm_case_section" does not exist
LINE 1: UPDATE crm_case_section SET code = '_DM' WHERE code = 'DM'
               ^

2016-09-20 08:30:19,786 11711 ERROR moto_migrated OpenUpgrade: relation "crm_case_section" does not exist
LINE 1: UPDATE crm_case_section SET code = '_DM' WHERE code = 'DM'
               ^
Traceback (most recent call last):
  File "/var/tmp/openupgrade/openupgradelib/openupgradelib/openupgrade.py", line 1066, in wrapped_function
    version)
  File "/var/tmp/openupgrade/8.0/addons/sales_team/migrations/8.0.1.0/pre-migration.py", line 37, in migrate
    cr.execute("UPDATE crm_case_section SET code = '_DM' WHERE code = 'DM'")
  File "/var/tmp/openupgrade/8.0/server/openerp/sql_db.py", line 158, in wrapper
    return f(self, *args, **kwargs)
  File "/var/tmp/openupgrade/8.0/server/openerp/sql_db.py", line 234, in execute
    res = self._obj.execute(query, params)
ProgrammingError: relation "crm_case_section" does not exist
LINE 1: UPDATE crm_case_section SET code = '_DM' WHERE code = 'DM'
               ^


```

**Comment by mgielissen** [comment]
_addons/sales_team/migrations/8.0.1.0/pre-migration.py_ seems to be a patch for demo data, but the database doesn't contain demo data. I comment out line 34 and the migration continues

**Comment by ovnicraft** [comment]
@mgielissen can you make a PR to fix this ?

**Comment by pedrobaeza** [comment]
No news, so closing.

**Comment by legalsylvain** [comment]
This has been fixed by #730
